rpcs3/Emu/Cell/Modules/cellSpurs.cpp

```diff
@@ -2926,13 +2919,9 @@ s32 cellSpursReadyCountAdd(ppu_thread& ppu, vm::ptr<CellSpurs> spurs, u32 wid, v
 		return CELL_SPURS_POLICY_MODULE_ERROR_STAT;
 	}
 
-	*old = vm::light_op(spurs->readyCount(wid), [&](atomic_t<u8>& v)
+	*old = vm::fetch_op(spurs->readyCount(wid), [&](u8& val)
 	{
-		return v.fetch_op([&](u8& val)
-		{
-			const s32 _new = val + value;
-			val = static_cast<u8>(std::clamp<s32>(_new, 0, 255));
-		});
+		val = static_cast<u8>(std::clamp<s64>(val + s64{value}, 0, 255));
```

Comment:
I think fw uses CMPWI instructions here for clamping, its ok for now, I'll check it later and worst case ill partially revert it.

Reply:
May need to emulate signed overflow without UB, or it's just impossible.

Reply:
Nvm, I think I fixed id.